mypy/server/aststrip.py

```diff
@@ -217,6 +217,9 @@ def is_duplicate_attribute_def(self, node: MemberExpr) -> bool:
         return any(info.get(node.name) is not None for info in self.type.mro[1:])
 
     def strip_ref_expr(self, node: RefExpr) -> None:
+        if node.fullname and (node.fullname.startswith('typing') or
+                node.fullname.startswith('builtins')):
+            return
```

Comment:
This doesn't seem right. It's possible that something that used to refer to a builtin will be updated to point to a user-defined definition after a refresh.

Reply:
This needs to be kept in sync since there are other places where `builtins` and `typing` are special-cased. If I remove this, one test will fail. Namely, if there is an alias like `A = Dict[str, int]`, then it is not reprocessed correctly.